internal/p/p.go

```diff
@@ -45,13 +54,14 @@ func NewCommand() *cobra.Command {
 
 	cmd.Flags().String("device-id", "", "device ID from 'spotify device list'")
 	cmd.Flags().String("playlist", "", "playlist name from 'spotify playlist list'")
+	cmd.Flags().String("album", "", "album name that you wish to play'")
 
 	return cmd
 }
 
-func p(api internal.APIInterface, query, contextQuery, deviceID string) (string, error) {
+func p(api internal.APIInterface, query, contextQuery, queryType, deviceID string) (string, error) {
 	if len(query) > 0 {
-		return play.Play(api, query, "", deviceID)
+		return play.Play(api, query, "", "track", deviceID)
 	}
```

Comment:
Here's what I'm thinking. If the user passes a `track`, `--album`, or `--playlist`, it's assumed that they want to play it (not pause it), so we should call `Play()` directly (you can use a `switch` here). The logic below these lines shouldn't change.